app/assets/javascripts/app/form-validation.js

```diff
@@ -21,7 +20,6 @@ const validate = {
   },
 
   addEvents() {
-    this.form.addEventListener('change', e => validateField(e.target));
```

Comment:
yes, is that a problem? If not, can we write a test for this? No tests failed when I removed this line so my assumption was that it's not critical behavior